client/signup/config/flows.js

```diff
@@ -87,7 +87,14 @@ const flows = {
 	},
 
 	main: {
-		steps: [ 'survey', 'design-type', 'themes', 'domains', 'plans', 'survey-user' ],
+		steps: [ 
+			'survey',
+			'design-type',
+			'themes',
+			'domains',
+			config.isEnabled( 'manage/plan-features' ) ? 'plansFeatures' : 'plans',
```

Comment:
Oh, sorry, I didn't understand you (and you were specific enough).

We've talked about this internally since and now I see the benefits of doing so.

Thanks. :+1: